projects/igniteui-angular/src/lib/grid/grid.component.spec.ts

```diff
@@ -419,10 +454,11 @@ describe('IgxGrid - input properties', () => {
                 for (let i = 0; i < headers.length; i ++) {
                     expect(headers[i].context.column.field).toEqual(fix.componentInstance.grid2.columns[i].field);
                 }
-                done();
-             });
+
```

Comment:
@tiliev Please leave done() callback which indicates that the test has finished.